### ISSUE TITLE
Questions about CLIP

### ISSUE THREAD

**Issue by steveguang** [UKPLab/sentence-transformers#1175]
Hi, do we have any tutorials on how to finetune CLIP? Besides, if I want to make the output dim of CLIP to be like 768, how should I do that? Thanks!

**Comment by nreimers** [comment]
Sadly there are no tutorial yet. But basically you use MultipleNegativesRankingLoss and you need to input one image and one text.

You could add a 512x768 dense layer on top of the CLIP model to make the dimension 768

**Comment by thtang** [comment]
Where to input input one image and one text pair for training (fine-tune)?

**Comment by krrishdholakia** [comment]
^
